src/components/node.js

```diff
@@ -339,7 +338,6 @@ class Node extends React.Component<INodeProps, INodeState> {
             />
           )}
           <use
-            data-index={index}
```

Comment:
removed this because it was used in prior code when v5 initially was developed, but it caused problems with selecting the right node.